_posts/2019-11-06-machine-readable-output.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Machine-readable Output Formats
+author: José Iván López González
+layout: post
+piwik: true
+---
+
+Third party programs and scripts (e.g., YaST) parse the output of the *snapper* cli commands to get some information. For example, to find out the number of all pre and post snapshots, scripts can run the following:
```

Comment:
Third-party (it is used as an adjective, see [Merriam Webster](https://www.merriam-webster.com/dictionary/third-party). Additionally, I would vote for upcasing *CLI*.